packages/shared/lib/services/nango-config.service.ts

```diff
@@ -366,7 +378,10 @@ export function convertV2ConfigObject(config: NangoConfigV2, showMessages = fals
                     webhookSubscriptions = [sync['webhook-subscriptions'] as string];
                 }
             }
+            const is_public = (isPublic !== undefined ? isPublic : sync.is_public === true) as boolean;
+            const pre_built = (isPublic !== undefined ? isPublic : sync.pre_built === true) as boolean;
```

Comment:
It was like that before but it's masking that isPublic can be null

Reply:
Updated with https://github.com/NangoHQ/nango/pull/1993/commits/ff41ddb750763966dbdbb237e91d4285429605b0